libs/components/forms/src/lib/modules/form-error/form-errors.component.html

```diff
@@ -42,6 +42,40 @@
       [errorText]="'skyux_form_error_date_max' | skyLibResources"
     />
 
+    <sky-form-error
+      *ngIf="errors['skyFuzzyDate'] && errors['skyFuzzyDate']['futureDisabled']"
+      errorName="fuzzyFutureDisabled"
+      [errorText]="
+        'skyux_form_error_fuzzy_date_future_disabled' | skyLibResources
+      "
+    />
+
+    <sky-form-error
+      *ngIf="errors['skyFuzzyDate'] && errors['skyFuzzyDate']['invalidDate']"
```

Comment:
I think this is supposed to be "invalid" instead of "invalidDate"? I'm not seeing this one show up